src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -40,10 +40,17 @@ object Arisa : ConfigSpec() {
 
         object Piracy : ModuleConfigSpec() {
             val piracyMessage by optional(
-                "You are currently using a *non-authorized* version of Minecraft." +
-                        " If you wish to purchase the full game, please visit the [Minecraft Store|https://minecraft.net/store].\r\n" +
-                        "We will not provide support for pirated versions of the game, these versions are modified and may contain malware."
-            )
+                "*Thank you for your report!*\n" +
+                        "However, this issue is {color:#FF5722}*Invalid*{color}.\n" +
+                        "\n" +
+                        "You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].\n" +
+                        "We will not provide support for pirated versions of the game, these versions are modified and may contain malware.\n" +
+                        "\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
+                        "\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]"
```

Comment:
Also `+` missing at the end of this line here.

Reply:
The end of the quick link line? None of those have +, should they?

Reply:
The I'm a bot message is after this.
Also you might want to add a `\n` after the quick links

Reply:
Pushed, although I feel like I've missed what you're asking me again (Sorry!)